cms/djangoapps/contentstore/views/videos.py

```diff
@@ -364,29 +473,29 @@ def videos_post(course, request):
     The returned array corresponds exactly to the input array.
     """
     error = None
-    if "files" not in request.json:
+    if 'files' not in request.json:
         error = "Request object is not JSON or does not contain 'files'"
```

Comment:
Yes this would most probably not be faced by user. 

But if someone tries to hack through front-end and sends empty request. This would be displayed to user. So, we can say only 5-10% chance, but this can be displayed to user.